requirements.txt

```diff
@@ -1,3 +1,4 @@
+augly>=0.1.10
```

Comment:
Is there an augly>0.1.10? I get errors when trying to pip install it. Looks like it only goes up to 0.1.9? https://pypi.org/project/augly/#history